src/newSeed/newSeed.html

```diff
@@ -1,14 +1,14 @@
 <template>
   <require from="../resources/elements/circledNumber/circledNumber.html"></require>
 
-  <div class="newSeedContainer ${currentStage.settings.stageNumber === 7 ? 'stage7' : ''}">
-    <div class="sideBarDropDownContainer" show.to-view="currentStage.settings.stageNumber === 7 ? false : true">
+  <div class="newSeedContainer ${currentStage.name}">
+    <div class="sideBarDropDownContainer  ${currentStage.name}">
```

Comment:
You should needn't this class.  Is enough to it have the class on the containing element